Dockerfile.desktop

```diff
@@ -0,0 +1,97 @@
+FROM ubuntu:lunar
+
+USER root
+
+# Set env
+ENV DEBIAN_FRONTEND=noninteractive \
+    MOZ_HEADLESS=1 \
+    LC_ALL=C.UTF-8 \
+    LANG=C.UTF-8 \
+    FIREFOX_VERSION=112.0 \
```

Comment:
Why so old? Per https://whattrainisitnow.com/,

VERSION | CHANNEL
----:|----
115| [ESR](https://whattrainisitnow.com/release/?version=esr)
120 | [Release](https://whattrainisitnow.com/release/?version=release)
121 | [Beta](https://whattrainisitnow.com/release/?version=beta)
122 | [Nightly](https://whattrainisitnow.com/release/?version=nightly)

And it also looks like tomorrow is release day, so numbers will bump again:
> _"Firefox 121 ships on December 19 (in 1 day)"_